wewatch-client/src/views/swiping/swiping-container.js

```diff
@@ -61,6 +64,12 @@ const SwipingContainer = (props) => {
     });
   });
 
+  function SwipingCompletedScreen () {
+    return(<Container><h1 style={{'text-align': "center", 'margin-top': '60px'}}>You've seen all potential movies for recommendation, please wait as your the others finish swiping!</h1></Container>)
+
+  }
+
+  if (!SwipingCompleted){
```

Comment:
I believe that this negates how our redirect strategy works, since the match/nomatch Redirect components cannot render if Swiping is completed. Could you try moving this conditional block to come after line 79?